pkgs/development/mobile/androidenv/cmdline-tools.nix

```
@@ -0,0 +1,41 @@
+{deployAndroidPackage, lib, package, autoPatchelfHook, makeWrapper, os, pkgs, pkgsi686Linux, stdenv, cmdLineToolsVersion, postInstall}:
+
+deployAndroidPackage {
+  name = "androidsdk";
+  inherit package os;
+  nativeBuildInputs = [ makeWrapper ]
+    ++ lib.optionals stdenv.isLinux [ autoPatchelfHook ];
+
+  patchInstructions = ''
+    ${lib.optionalString (os == "linux") ''
+      # Auto patch all binaries
+      autoPatchelf .
+    ''}
+
+    # Strip double dots from the root path
+    export ANDROID_SDK_ROOT="$out/libexec/android-sdk"
+
+    # Wrap all scripts that require JAVA_HOME
+    for i in $ANDROID_SDK_ROOT/cmdline-tools/${cmdLineToolsVersion}/bin; do
+      find $i -maxdepth 1 -type f -executable | while read program; do
```

```suggestion
      find $ANDROID_SDK_ROOT/cmdline-tools/${cmdLineToolsVersion}/bin -maxdepth 1 -type f -executable | while read program; do
```
Would this do the same?

---

Eagle eyes! good job and thank you. Fixed [a19f22c](https://github.com/NixOS/nixpkgs/pull/208793/commits/a19f22c9c96f73e903099f8a825f500b907d6862)